Content.Shared/Slippery/SlipperySystem.cs

```diff
@@ -52,6 +56,18 @@ private static void OnNoSlipAttempt(EntityUid uid, NoSlipComponent component, Sl
         args.Cancel();
     }
 
+    private void OnThrownSlipAttempt(EntityUid uid, ThrownItemComponent comp, SlipEvent args)
+    {
+        args.Cancel();
+
+        //TODO: If it's after 2024 March and this popup is still here, remove it
+        // Only here so people who don't read even the changelog won't think soap suddenly broke
+        // Can't be popupclient because the goal is to let every onlooker know that this was not a glitch or "omg noslips"
+        _popup.PopupEntity(Loc.GetString("thrown-slippery-missed"), uid);
+
+        return;
+    }
+
```

Comment:
Returning from void statements at the end is non-idiomatic C# so remove this.
```suggestion
    }

```